app/views/auctions/_winning_bid.html.erb

```diff
@@ -1,20 +1,4 @@
-<% if defined? for_winners_page %>
+<div class="issue-bids-info-item">Bidding is closed</div>
 
-<% else %>
-  <div class="issue-bids-info-item">Bidding is closed</div>
-<% end %>
-
-<% if auction.bids? %>
-  <div class="issue-bids-info-item"><% if defined? for_winners_page %><% else %>
-    <span>Winning bid: </span>
-  <% end %><%= auction.highlighted_bid_amount_as_currency %></div>
-
-<% else %>
-  <div class="issue-bids-info-item">No bids</div>
-<% end %>
-
-<% unless AuctionStatus.new(auction).over? or defined? for_winners_page %>
-  <span class="issue-bids-info-item bid-deadline">
-    Bid start time: <%= auction.relative_start_time %>
-</span>
-<% end %>
+<%= render partial: auction.winning_bid_partial, locals: { auction: auction } %>
+<%= render partial: auction.start_time_partial, locals: { auction: auction } %>
```

Comment:
yes! I think this polymorphic partial stuff is the one thing that actually looks better after this PR